app/presenters/scoped_events_presenter.rb

```diff
@@ -0,0 +1,34 @@
+class ScopedEventsPresenter
+  attr_reader :event_feed, :request
+
+  def initialize(event_feed, request)
+    @event_feed = event_feed
+    @request = request
+  end
+
+  def as_json(opt = {})
+    {
+      _meta: PaginationPresenter.new(event_feed.events, [], request),
+      records: event_feed.events.map { |e| event_with_comments(e) }
+    }
+  end
+
+  private
+  def event_with_comments(event)
+    event_json = EventPresenter.new(event).as_json
+    if event.action == 'commented'
```

Comment:
For "commented on" actions, include the comment in addition to all the normal event fields.